src/provider/azureProvider.ts

```diff
@@ -95,7 +84,7 @@ export default class AzureProvider {
       .on('error', () => {
         console.error('Disconnected from log streaming.');
       })
-      .on('end', this.getLogsStream.bind(this))
+      .on('end', () => this.getLogsStream(functionName))
```

Comment:
Advantage of arrow function over bind? Should we be using arrow function throughout?